src/main/java/org/jabref/logic/util/URLUtil.java

```diff
@@ -83,9 +84,9 @@ public static String cleanGoogleSearchURL(String url) {
      */
     public static boolean isURL(String url) {
         try {
-            create(url);
```

Comment:
why no reuse of the existing methods? Did you read `createUri(String url)`

You should try to train yourself to be able to read code. This class is NOT calling other JabRef classes, it is on its own.